x-pack/plugins/enterprise_search/public/applications/app_search/components/engines/engines_overview.tsx

```diff
@@ -100,15 +112,27 @@ export const EnginesOverview: React.FC = () => {
           />
         </EuiPageContentBody>
 
-        {metaEngines.length > 0 && (
+        {canViewMetaEngines && (
           <>
             <EuiSpacer size="xl" />
             <EuiPageContentHeader>
-              <EuiTitle size="s">
-                <h2>
-                  <MetaEngineIcon /> {META_ENGINES_TITLE}
-                </h2>
-              </EuiTitle>
+              <EuiPageContentHeaderSection>
+                <EuiTitle size="s">
+                  <h2>
+                    <MetaEngineIcon /> {META_ENGINES_TITLE}
+                  </h2>
+                </EuiTitle>
+              </EuiPageContentHeaderSection>
+              <EuiPageContentHeaderSection>
+                <EuiButtonTo
+                  color="primary"
+                  fill
+                  data-test-subj="appSearchEnginesMetaEngineCreationButton"
+                  to={META_ENGINE_CREATION_PATH}
+                >
+                  {CREATE_A_META_ENGINE_BUTTON_LABEL}
```

Comment:
[not a change request, just a general thought/some feedback] I'd lean towards keeping i18n var names more abstract/generic and less tied to the actual copy if possible, it feels more future-proof in the event that copy changes. In this case I'd have dropped the `a` here, e.g.: just `CREATE_META_ENGINE_BUTTON_LABEL` or even `META_ENGINE_CREATION_BUTTON_LABEL`

Again not a change request since I think the other create button has an `an` also, mostly just voicing a generality